.github/copilot-instructions.md

```
@@ -0,0 +1,508 @@
+# Copilot / Automation Workflow Guidance
+
+START OF EVERY SESSION / CHANGE (NON‑NEGOTIABLE):
+1. OPEN and READ this file (`.github/copilot-instructions.md`) in full before performing ANY action (search, edit, build, test, benchmark, or commit). 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)